examples/tutorials/Tutorial_9_neural_networks.ipynb

```diff
@@ -0,0 +1,1146 @@
+     "exception": false,
+     "start_time": "2021-06-22T20:10:53.158298",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Standard python libraries\n",
+    "import os\n",
+    "import time\n",
+    "\n",
+    "# Essential DS libraries\n",
+    "import optuna\n",
+    "import requests\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from sklearn.metrics import roc_auc_score\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "import torch\n",
+    "from copy import deepcopy as copy\n",
+    "import torch.nn as nn\n",
+    "from collections import OrderedDict\n",
+    "\n",
+    "# LightAutoML presets, task and report generation\n",
+    "from lightautoml.automl.presets.tabular_presets import TabularAutoML\n",
+    "from lightautoml.tasks import Task"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "486dff3d",
+   "metadata": {
+    "papermill": {
+     "duration": 0.064234,
+     "end_time": "2021-06-22T20:11:01.619010",
+     "exception": false,
+     "start_time": "2021-06-22T20:11:01.554776",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "source": [
+    "### 0.2. Constants\n",
+    "\n",
+    "Here we setup the constants to use in the kernel:\n",
+    "- `N_THREADS` - number of vCPUs for LightAutoML model creation\n",
+    "- `N_FOLDS` - number of folds in LightAutoML inner CV\n",
+    "- `RANDOM_STATE` - random seed for better reproducibility\n",
+    "- `TEST_SIZE` - houldout data part size \n",
+    "- `TIMEOUT` - limit in seconds for model to train\n",
+    "- `TARGET_NAME` - target column name in dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "64dfd5d0",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-06-22T20:11:01.758476Z",
+     "iopub.status.busy": "2021-06-22T20:11:01.757403Z",
+     "iopub.status.idle": "2021-06-22T20:11:01.760870Z",
+     "shell.execute_reply": "2021-06-22T20:11:01.760168Z",
+     "shell.execute_reply.started": "2021-06-22T19:06:51.523697Z"
+    },
+    "papermill": {
+     "duration": 0.077787,
+     "end_time": "2021-06-22T20:11:01.761030",
+     "exception": false,
+     "start_time": "2021-06-22T20:11:01.683243",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "N_THREADS = 4\n",
+    "N_FOLDS = 5\n",
+    "RANDOM_STATE = 42\n",
+    "TEST_SIZE = 0.2\n",
+    "TIMEOUT = 300\n",
+    "TARGET_NAME = 'TARGET'\n",
+    "\n",
+    "np.random.seed(RANDOM_STATE)\n",
+    "torch.set_num_threads(N_THREADS)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "62e42740",
+   "metadata": {},
+   "source": [
+    "### 0.3. Data loading"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "b8c3218d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATASET_DIR = '../data/'\n",
+    "DATASET_NAME = 'sampled_app_train.csv'\n",
```

Comment:
remove redundant variables DATASET_NAME and DATASET_DIR -- they are used just only once